### PR TITLE
Add `preview-maid` template

### PR DESCRIPTION
**Pull request by fletchto99**
Preview maid is a script I've created that connects to a user's plex instance and goes over each library with thumbnail generation enabled and finds any files which are missing previews. By default it runs on a schedule daily at `00:00`. An optional `RUN_ONCE` environment variable can be passed to have it run right at startup and output the results immediately.

Preview maid's repo can be found here: https://github.com/fletchto99/preview-maid

### CODE REVIEW

File: templates/preview-maid.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/fletchto99/preview-maid</Project>
+  <Overview>Preview maid is a tool to help you find missing thumbnail previews in your plex library. By default it will run upon start and then on a schedule daily at 00:00.</Overview>
+  <Category>Tools:Utilities Status:Stable</Category>
+  <TemplateURL>https://github.com/Kometa-Team/Unraid-Templates/blob/master/ImageMaid.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/preview-maid.png</Icon>
+  <Config Name="PLEX_URL" Target="PLEX_URL" Default="" Mode="" Description="The URL to your Plex instance" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="PLEX_TOKEN" Target="PLEX_TOKEN" Default="" Mode="" Description="Your plex token" Type="Variable" Display="always" Required="true" Mask="true"/>
+  <Config Name="RUN_ONCE" Target="RUN_ONCE" Default="False" Mode="" Description="Set to true if you wish for only on run after which the container will terminate." Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline templates/preview-maid.xml:17]:
```suggestion
  <Config Name="RUN_ONCE" Target="RUN_ONCE" Default="False|True" Mode="" Description="Set to true if you wish for only on run after which the container will terminate." Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: templates/preview-maid.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <Repository>fletchto99/preview-maid</Repository>
+  <Registry>https://hub.docker.com/r/fletchto99/preview-maid</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/fletchto99/preview-maid</Project>
+  <Overview>Preview maid is a tool to help you find missing thumbnail previews in your plex library. By default it will run upon start and then on a schedule daily at 00:00.</Overview>
+  <Category>Tools:Utilities Status:Stable</Category>
+  <TemplateURL>https://github.com/Kometa-Team/Unraid-Templates/blob/master/ImageMaid.xml</TemplateURL>
```
Comment (nwithan8) [inline templates/preview-maid.xml:13]: This needs updating. Is this a fork of ImageMaid?

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/preview-maid.xml (hunk 2 above)
Comment (fletchto99) [reply templates/preview-maid.xml:13]: Ah good catch. No it's not - I just used their xml as a template

File: templates/preview-maid.xml (hunk 1 above)
Comment (nwithan8) [reply templates/preview-maid.xml:17]: I tend to prefer making enum-based config variables "required" to ensure the user always provides one of the enums (typically the default - the first in the pipe-delimited list)

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**